package/kedro_viz/launchers/cli.py

```diff
@@ -158,7 +166,7 @@ def run(
             "extra_params": params,
         }
         if autoreload:
-            project_path = Path.cwd()
+            project_path = Path(project_path) if project_path else Path.cwd()
```

Comment:
run_server also does `Path(project_path) if project_path else Path.cwd()` on line 109 `server.py`, so we can just pass project_path and this is not needed